backend/app/api/ws/local_executor_namespace.py

```
@@ -0,0 +1,703 @@
+                    task_json["status"]["updatedAt"] = datetime.now().isoformat()
+                    task_json["status"]["completedAt"] = datetime.now().isoformat()
+                task.json = task_json
+                task.updated_at = datetime.now()
+                flag_modified(task, "json")
+
+            db.commit()
+            logger.info(
+                f"[LocalExecutor] Updated task/subtask status: task_id={task_id}, "
+                f"subtask_id={subtask_id}, status={status}"
+            )
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)